### DISCUSSION TITLE
Bigger Play Button

### DISCUSSION THREAD

**Discussion by DJDukeSr** [AzuraCast/AzuraCast#6765] [Q&A]
Can somebody help me to increase the size of the play button, so it will bigger on my website. Thanks in Advance.
I searched the article before but I do not understand what and where to program it.  I am not a dev, but if explained simple I might follow the a-z instruction. Thanks

**Comment by DJDukeSr** [comment]
I think there nobody who could help

**Comment by gAlleb** [comment]
Hi @DJDukeSr 
1. Navigate to `Administration` — `Custom Branding` — `Custom CSS for Public Pages`

<img width="250" alt="" src="https://github.com/AzuraCast/AzuraCast/assets/144447266/35b3b6a5-48e9-434c-9d81-ac6dc616522d">

2. Paste css code with desired font-size, e.g.

```
.radio-player-widget .radio-controls .radio-control-play-button .icon {
  font-size: 3.25rem;
}
```

3. Save. The same result but only for 1 station (if you have few) may be achieved by navigating to `Station Profile` — `Branding` — `Custom CSS for Public Page`.

**Comment by DJDukeSr** [comment]
Thanks Stefan, send you feed back when executed.

Daniel John Ducret
Brückenstrasse 31
3005 Bern
Switzerland

***@***.******@***.***>
t: +41 78 733 0000

linkedin<https://www.linkedin.com/in/daniel-john-ducret-3858a954/>

[cid:F2096E93-596B-4FE9-ACA0-267308D5772B]

Am 17.12.2023 um 06:40 schrieb Stefan ***@***.******@***.***>>:



Hi @DJDukeSr<https://github.com/DJDukeSr>

  1.  Navigate to "Administration" — "Custom Branding" — "Custom CSS for Public Pages"

2023-12-17.08.34.09.png (view on web)<https://github.com/AzuraCast/AzuraCast/assets/144447266/35b3b6a5-48e9-434c-9d81-ac6dc616522d>

  1.  Paste css code with desired font-size, e.g.

.radio-player-widget .radio-controls .radio-control-play-button .icon {
  font-size: 3.25rem;
}


  1.  Save. The same result but only for 1 station (if you have few) may be achieved by navigating tp "Station Profile" — "Branding" — "Custom CSS for Public Pages".

**Reply by DJDukeSr** [reply]
It's working. Thank you very much Stefan.
